frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -60,6 +69,28 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   const replica = isArbiterEnabled ? OCS_DEVICE_SET_ARBITER_REPLICA : OCS_DEVICE_SET_REPLICA;
   const name = getName(ocsConfig);
 
+  React.useEffect(() => {
```

Comment:
Its okay to show the error on clicking add capacity button for now. 
Since otherwise , we need to disable add capacity button and change the error message text as well.
For future I want to move to use FormSelect in this modal and run validations then 
https://www.patternfly.org/v4/components/form-select#invalid. Hence lets keep